decidim-core/lib/decidim/form_builder.rb

```diff
@@ -30,7 +30,7 @@ def translated(type, name, options = {})
         tabs_panels = content_tag(:ul, class: "tabs", id: "#{name}-tabs", data: { tabs: true }) do
           locales.each_with_index.inject("".html_safe) do |string, (locale, index)|
             string + content_tag(:li, class: tab_element_class_for("title", index)) do
-              title = I18n.with_locale(locale){ I18n.t(locale, scope: "locales") }
+              title = I18n.with_locale(locale){ I18n.t("name", scope: "locale") }
```

Comment:
You will need to change this in the rspec helper and the `LanguageChooserHelper` too.